src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorRequest.cs

```diff
@@ -32,6 +32,11 @@ internal SplunkEvent(string logEvent, string source, string sourceType, string h
                 jsonPayLoad = jsonPayLoad + @",""index"":""" + index + @"""";
```

Comment:
We are using `ConcurrentQueue`.  I understand it has a some overhead.  @glennblock, any chance you guys are going to port to dotnet core?